CONTRIBUTING.md

```diff
@@ -198,6 +187,61 @@ and leave it running (for manual verification).
    Your Conjur server is running!
    ```
 
+1. Configure Conjur
+
+   Create an initial `default` account. Refer to the instructions here:
+   <https://www.conjur.org/get-started/install-conjur.html#install-and-configure>
+
+   ```
+   export POD_NAME=$(kubectl get pods \
+          -l "app=conjur-oss" \
+          -o jsonpath="{.items[0].metadata.name}")
+   kubectl exec $POD_NAME --container=conjur-oss conjurctl account create default
+   ```
+
+   Note that the above commands give you the public key and admin API key for
+   the account that was created. Back them up in a safe location.
+
+1. Connect to Conjur
+
+   Start a container with Conjur CLI and authenticate with Conjur as
+   user `admin`:
+
+   ```
+   INGRESS_SVC=$(kubectl get svc --no-headers -o custom-columns=":metadata.name" | grep conjur-oss-ingress)
+   export EXT_IP=$(kubectl get svc $INGRESS_SVC -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+   docker run --rm -it --env EXT_IP --env CERTIFICATE_CN --entrypoint bash cyberark/conjur-cli:5
+   ```
+
+   In the Conjur CLI container, set up an entry in `/etc/hosts` for the
+   Conjur server. (Alternatively, you can configure a DNS A record mapping
+   the hostname that you configured in $CERTIFICATE_CN to the Conjur's external IP address).
+
+   ```
+   grep -q $CERTIFICATE_CN /etc/hosts && \
+       sed -i "s/.*$CERTIFICATE_CN/$EXT_IP $CERTIFICATE_CN/" /etc/hosts || \
+       echo "$EXT_IP $CERTIFICATE_CN" >> /etc/hosts
+   ```
+
+   Then connect with the Conjur server using the account just created:
+
+   ```
+   conjur init -u https://$CERTIFICATE_CN -a default
+   ```
+
+   And then authenticate as user `admin`, using the admin API key obtained
+   in the previous section as a password:
+
+   ```
+   conjur authn login -u admin -p <API_KEY>
+   ```
+
+   Finally, check that you are logged in as user `admin`:
```

Comment:
Not a blocker but maybe we should show what the output should look like?

Reply:
Yes, good point. It's pretty important to know whether this worked or not.